src/main.py

```diff
@@ -10,11 +10,23 @@
 # pylint: enable=no-name-in-module
 
 
-def main(argv):
+def main() -> None:
     """
     Program entry point.
     """
-    download_stdlib(argv)
+    arg_parser = argparse.ArgumentParser(
+        description="Generates documentation sites for Enso Standard Library."
+    )
+    arg_parser.add_argument("token", help="GitHub user Personal Access Token.")
+    args = arg_parser.parse_args()
+    run_gen(args.token)
+
+
+def run_gen(token: str) -> None:
+    """
+    Runs the actual generation process.
+    """
+    download_stdlib(token)
```

Comment:
Put the contents of `run_gen` back in `main`, and move the argparse logic into the `if __name__ == "__main__"` block.